test/hotspot/jtreg/runtime/ClassFile/OOMCrashClass4000_1.java

```diff
@@ -0,0 +1,111 @@
+import org.objectweb.asm.AnnotationVisitor;
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.FieldVisitor;
+import org.objectweb.asm.Label;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.RecordComponentVisitor;
+
+/*
+    This is class is a dumper for the original OOMCrashClass4000_1.class, i.e. its dump() method
+    produces a byte array with the original class file data.
+
+    To get this source code, one needs to run the following command:
+    java jdk.internal.org.objectweb.asm.util.ASMifier OOMCrashClass4000_1.class >> OOMCrashClass4000_1.java
+
+    The resulting java source code is large (>2 mb), so certain refactoring is applied.
+ */
+
+public class OOMCrashClass4000_1 implements Opcodes {
+
+    public static byte[] dump() throws Exception {
+
+        ClassWriter classWriter = new ClassWriter(0);
+        FieldVisitor fieldVisitor;
+        RecordComponentVisitor recordComponentVisitor;
+        MethodVisitor methodVisitor;
+        AnnotationVisitor annotationVisitor0;
+
+        classWriter.visit(V1_1, ACC_PUBLIC | ACC_SUPER, "OOMCrashClass4000_1", null, "java/applet/Applet", null);
+
+        classWriter.visitSource("<generated>", null);
+
+        {
+            methodVisitor = classWriter.visitMethod(ACC_PUBLIC | ACC_STATIC, "main0", "([Ljava/lang/String;)V", null, null);
+            methodVisitor.visitCode();
+            Label label0 = new Label();
+            methodVisitor.visitLabel(label0);
+            methodVisitor.visitVarInsn(ALOAD, 0);
+            methodVisitor.visitInsn(ICONST_0);
+            methodVisitor.visitInsn(AALOAD);
+            methodVisitor.visitMethodInsn(INVOKESTATIC, "java/lang/Integer", "parseInt", "(Ljava/lang/String;)I", false);
+            methodVisitor.visitVarInsn(ISTORE, 1);
+            Label label1 = new Label();
+            methodVisitor.visitJumpInsn(GOTO, label1);
+            Label label2 = new Label();
+            methodVisitor.visitLabel(label2);
+            methodVisitor.visitInsn(RETURN);
+
+
```

Comment:
In this extra line, can you add a comment that this is the line that overflows the jump target for the jsr?